test/pom.xml

```diff
@@ -63,7 +63,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.13</version>
+      <version>2.19-20170218.174156-1</version>
```

Comment:
https://github.com/jenkinsci/jenkins-test-harness/pull/48 is what we need for this